app/components/layouts/chat-header.tsx

```diff
@@ -26,9 +26,13 @@ export function ChatHeader({
         <div className="flex items-center gap-4">
           <Avatar className="w-14 h-14">
             <AvatarImage src={image} />
-            <AvatarFallback className="bg-gray-300">
-              {getInitials(name)}
-            </AvatarFallback>
+            {name && name.toLowerCase() !== 'loading...' ? (
```

Comment:
I have a very loose preference for wrapping this in parens, but not a blocker

```suggestion
            {name && (name.toLowerCase() !== 'loading...') ? (
```